templates/navigation-bar.twig

```diff
@@ -60,9 +60,12 @@
 			<span class="screen-reader-text">{{ __( 'Toggle search form', 'planet4-master-theme' ) }}</span>
 		</button>
 		<form id="search_form" action="{{ data_nav_bar.home_url }}" class="form nav-item nav-search-wrap">
-			<input type="text" class="form-control" placeholder="{{ search_label }}"
+			<input id="search_input" type="text" class="form-control" placeholder="{{ search_label }}"
 				   value="{{ data_nav_bar.search_query|e('wp_kses_post')|raw }}" name="s" aria-label="Search">
 			<input id="orderby" type="hidden" name="orderby" value="{{ selected_sort ?? default_sort }}"/>
+			{% if true == is_elastic_search %}
```

Comment:
This is probably a leftover from a previous commit, I don't see it used anywhere.